pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/misc/MultiPlayerShowcase.kt

```diff
@@ -30,13 +35,40 @@ import ch.srgssr.pillarbox.demo.ui.theme.paddings
  */
 @Composable
 fun MultiPlayerShowcase() {
-    val multiPlayerViewModel: MultiPlayerViewModel = viewModel()
     var swapLeftRight by remember {
         mutableStateOf(false)
     }
-    val playerLeft = multiPlayerViewModel.getPlayerLeft(swapLeftRight)
-    val playerRight = multiPlayerViewModel.getPlayerRight(swapLeftRight)
-
+    val context = LocalContext.current
+    val playerOne = remember {
+        PlayerModule.provideDefaultPlayer(context).apply {
+            repeatMode = Player.REPEAT_MODE_ONE
+            setMediaItem(DemoItem.LiveVideo.toMediaItem())
+            prepare()
+        }
+    }
+    val playerTwo = remember {
+        PlayerModule.provideDefaultPlayer(context).apply {
+            repeatMode = Player.REPEAT_MODE_ONE
+            setMediaItem(DemoItem.DvrVideo.toMediaItem())
+            prepare()
+        }
+    }
+    DisposableEffect(Unit) {
+        onDispose {
+            playerOne.release()
+            playerTwo.release()
+        }
+    }
+    LifecycleResumeEffect(Unit) {
+        playerOne.play()
+        playerTwo.play()
+        onPauseOrDispose {
+            playerOne.pause()
+            playerTwo.pause()
+        }
+    }
+    val leftPlayer = if (swapLeftRight) playerOne else playerTwo
+    val rightPlayer = if (swapLeftRight) playerTwo else playerOne
```

Comment:
Just a detail, but shouldn't this be:
```suggestion
    val leftPlayer = if (swapLeftRight) playerTwo else playerOne
    val rightPlayer = if (swapLeftRight) playerOne else playerTwo
```